src/pykiso/cli.py

```diff
@@ -90,12 +90,40 @@ def check_file_extension(
     return paths
 
 
+class CommandWithOptionalFlagValues(click.Command):
+    def parse_args(self, ctx, args):
+        """Translate any flag `--junit=value` as flag `--junit` with changed flag_value=value"""
+        # filter flags
+        flags = [
+            flag
+            for flag in self.params
+            if isinstance(flag, click.Option)
+            and flag.is_flag
+            and not isinstance(flag.flag_value, bool)
+        ]
+        junit_prefixes = {
```

Comment:
Could this somehow be made generic? 
It feels weird to have a class named CommandWithOptionalFlagValues (very generic name) that only handles one specific flag name --junit